src/nvim/spell.c

```diff
@@ -4542,9 +4525,10 @@ static void suggest_trie_walk(suginfo_T *su, langp_T *lp, char_u *fword, bool so
       if (c2 != NUL && TRY_DEEPER(su, stack, depth, SCORE_SWAP)) {
         go_deeper(stack, depth, SCORE_SWAP);
 #ifdef DEBUG_TRIEWALK
-        sprintf(changename[depth], "%.*s-%s: swap %c and %c",
-            sp->ts_twordlen, tword, fword + sp->ts_fidx,
-            c, c2);
+        snprintf(changename[depth], sizeof changename[0],
```

Comment:
Missing parenthesis: while `sizeof` being operator can be used without them it is always written like if it was a function.

Reply:
Done. 
Didn't find this rule in the relevant style guide section, https://neovim.io/develop/style-guide.xml#sizeof. If it is indeed not listed, should it be added?